backend/src/main/java/gov/cdc/usds/simplereport/logging/PatientExperienceLoggingInterceptor.java

```diff
@@ -52,4 +52,8 @@ public void afterCompletion(
     log.trace("Final logging cleanup step.");
     MDC.remove(LoggingConstants.REQUEST_ID_MDC_KEY);
   }
+
+  private String sanitizeRequestURI(String requestURI) {
+    return requestURI.matches("\\w*") ? requestURI : "request contained invalid characters";
```

Comment:
to confirm, we don't want to sanitize the original request URI just display a different msg if there are unsafe characters?

Reply:
Yep! Because we won't display the unsafe characters in the log regardless, so it's more helpful to know that there were invalid characters (otherwise the log wouldn't reflect the attempt at all).